### DISCUSSION TITLE
[question] Zim-Wiki on Windows 11

### DISCUSSION THREAD

**Discussion by GottlobFrege** [zim-desktop-wiki/zim-desktop-wiki#1850] [Q&A]
Hello,

first of all, I must say that unfortunately I have no idea about IT, programming and so on. I am a normal Windows user who can use the standard programs in a normal way. (And this is my first post here, please forgive me if question is in wrong place.)

I have to thank the creator of ZimWiki. I've been using it for months, and on a daily basis. And it is exactly what I always needed and wanted. I am studying philosophy in Germany as part of a master's program. Over the years I've accumulated a huge amount of notes, citations, outlines, summaries, ideas, etc. in many different Word documents, and I had long lost track of them all. Transferring all of this to ZimWiki took me a lot of time, but it was definitely worth it. Now I have all my notes in a single wiki, with many extremely helpful cross-reference-links, and everything is globally searchable. A dream! 

But: now I'm afraid to upgrade to Windows 11. Will ZimWiki work on it?

**Comment by jaap-karssenberg** [comment]
Personally I have not yet tested it on windows 11, waiting to hear from
users that have done so

**Comment by tridian-tn** [comment]
@GottlobFrege although I haven't rigorously tested it, I've been running Zim on Windows 11 since I upgraded from Win10 in mid-December. No noticeable issues so far 🤞

https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/1639 continues to be a problem without the toolbar setting change also mentioned in that issue, but workaround means I no longer see it.

**Comment by shivams** [comment]
I can report a few minor issues, but they are mostly related to the GTK toolkit. But maybe some can be fixed by Zim.

The issues are regarding a feeling of "non-native" integration with the Windows 11 window management system. E.g. pressing the key `Win` + `up/down/right/left arrow` doesn't move the Zim window at all. It's a smooth way of snapping the windows in Windows 11.

One major issue that comes out of this non-native feel is that if I open multiple Zim notebooks, then sometimes I am not at all able to switch to some particular notebook. I have to make that notebook the default notebook and then only I can actually open and see it. I'm not sure why this issue happens. It doesn't happen on all notebooks. I have about 6 notebooks, and it happened on 2 of them. If I make such "faulty" notebooks default at least once, they open just fine in the future, even when they are not default.

**Reply by introt** [reply]
Regarding window management, please see #2007

**Comment by Statisticonomicon** [comment]
I have been running it fine on Win11 until today, I run it, it shows in the bottom bar but I cannot get to it when I click on it, it has been problematic from the start with frequent crashes when I try to collapse categories or expand  them, when trying to switch to another language font, it is generally bad on Windows, everything compiled pre Win11 is. It runs beautifully on my Linux machine. Maybe I should get a lobotomy so I can switch to OneNote.
